tests/wicked/before_test.pm

```diff
@@ -43,6 +45,32 @@ sub run {
     file_content_replace('/etc/sysconfig/network/config', '--sed-modifier' => 'g', '^WICKED_DEBUG=.*' => 'WICKED_DEBUG="all"', '^WICKED_LOG_LEVEL=.*' => 'WICKED_LOG_LEVEL="debug2"');
     #preparing directories for holding config files
     assert_script_run('mkdir -p /data/{static_address,dynamic_address}');
+
+    if (systemctl('is-active NetworkManager', ignore_failure => 1) == 0) {
```

Comment:
just in case. inside the `if` please add `die "wicked and NM enabled simultaneously " if systemctl('is-active wicked', ignore_failure => 1) == 0;`

Reply:
Can this situation happen at all?

Reply:
I would like to extract this into a function inside this test module. Just to not blow this `run()` more and more.